chapters/appendix-VI-external-repository-identifiers.md

```diff
@@ -14,59 +14,68 @@ There are currently four defined categories:
 The following sections provide details on the available types
 and the locator formats for each of the categories.
 
-
 ## Security
 
 ### cpe22Type <a name="cpe22"></a>
 
 Locator Format:
 
-    "[c][pP][eE]:/[AHOaho]?(:[A-Za-z0-9\._\-~%]*){0,6}"
+```text
+"[c][pP][eE]:/[AHOaho]?(:[A-Za-z0-9\._\-~%]*){0,6}"
+```
 
 Contextual Example:
 
-    cpe:/o:canonical:ubuntu_linux:10.04:-:lts
+```text
+cpe:/o:canonical:ubuntu_linux:10.04:-:lts
+```
 
 External Reference Site: [https://nvd.nist.gov/cpe](https://nvd.nist.gov/cpe)
 
 Documentation: [https://cpe.mitre.org/files/cpe-specification_2.2.pdf](https://cpe.mitre.org/files/cpe-specification_2.2.pdf)
 
-
 ### cpe23Type <a name="cpe23"></a>
 
 Locator Format:
 
-    "cpe:2\.3:[aho\*\­]
-    (:(((\?*|\*?)([a­zA­Z0­9\­\._]|(\\[\\\*\?!
-    "#$$%&'\(\)\+,/:;<=>@\[\]\^`\{\|}~])
-    )+(\?*|\*?))|[\*\­])){5}
-    (:(([a­zA­Z]{2,3}(­([a­zA­Z]{2}|[0­9]{3
-    }))?)|[\*\­]))
-    (:(((\?*|\*?)([a­zA­Z0­9\­\._]|(\\[\\\*\?!
-    "#$$%&'\(\)\+,/:;<=>@\[\]\^`\{\|}~])
-    )+(\?*|\*?))|[\*\­])){4}"
+```text
+"cpe:2\.3:[aho\*\­]
```

Comment:
Same question as above, for this regex should the start and end quotation marks (") be removed?